packages/@webex/plugin-cc/src/AgentProfileService/constants.ts

```diff
@@ -0,0 +1,4 @@
+// making query params configurable for List Teams and List Aux Codes API
+export const page = 0;
```

Comment:
Please keep constants in CAPS as in our other plugins like calling.
Also, I think we can rename them as default page, default page size etc

Reply:
**Code Corrected!**